src/app/runtime/wac/services/powershell-service.ts

```diff
@@ -119,9 +119,14 @@ export class PowershellService {
       }),
       catchError((e, _) => {
         let rethrow = e;
-        // WAC wrap the powershell error message around this AjaxError object. We would unwrap it for easier readability
-        if (e.name == "AjaxError" && e.status == 400 && e.response.error) {
-          rethrow = e.response.error;
+        // WAC wrap the powershell error (or exception) message around this AjaxError object. We would unwrap it for easier readability
+        if (e.name == "AjaxError" && e.status == 400) {
+          if (e.response.error && !e.response.exception) {

```

Comment:
Now we are catching exception, this break the logic in `runtime.wac.ts` `GetApiKey`. Update accordingly